cylc/flow/task_job_mgr.py

```diff
@@ -912,7 +911,13 @@ def _run_job_cmd(
 
         # Go through each list of itasks and carry out commands as required.
         for platform_name, itasks in sorted(auth_itasks.items()):
-            platform = get_platform(platform_name)
+            try:
```

Comment:
Here I think that the correct choice is to try this and move on logging an error if the command fails.

We might want "try N times" or "try until we get the response we want", or some combo of the two, but I think that needs to be handled separately.

Reply:
Worth handling the two separately?